src/components/Molecules/Task/story.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Component from "./index";
+
+export default {
+  component: Component,
+  title: "Molecules/Task",
+  parameters: {
+    backgrounds: {
+      default: "dark",
+    },
+  },
+};
+
+const Template = (args) => <Component {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  onEditComplete: () => console.log("taskname changed ${taskName}"),
```

Comment:
```suggestion
  onEditComplete: (taskName) => console.log(`taskname changed ${taskName}`),
```
文字列内部で変数を参照したいときは バッククオート `のテンプレートリテラルでいきましょう！
また、変更後のタスク名を表示するためには引数として受け取れるようにしておくのも必要ですね。、